spdx-utils/src/main/kotlin/SpdxLicense.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2017-2019 HERE Europe B.V.
+ * Copyright (C) 2017-2020 HERE Europe B.V.
```

Comment:
Wanna add Bosch?

Reply:
Well, that code is auto-generated, and the generator was written when I was still at HERE, so I think it's fine.